extensions/answer_summarizers/models_test.py

```diff
@@ -40,6 +40,17 @@ class CalculationUnitTestBase(test_utils.GenericTestBase):
     def _create_answer_dict(
             self, answer, time_spent_in_card=3.2, session_id='sid1',
             classify_category=exp_domain.EXPLICIT_CLASSIFICATION):
+        """Returns the answer object.
```

Comment:
It returns a dict right? object doesn't sound right here.

Reply:
Done